handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -494,6 +494,12 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         }
         try {
             wrap(ctx, false);
+        } catch (Exception e) {
+            // Just fail the pending writes in case of an SSLException. If there are no pending writes rethrow.
+            if (pendingUnencryptedWrites.removeAndFailAll(e)) {
```

Comment:
@Scottmitch I think we always want to propagate the exception if `pendingUnencryptedWrites.removeAndFailAll(...)` returns false. As otherwise we will swallow it which I think is not good. So we will at least catch it in the `AbstractChannelHandlerContext` and log it. So I am not sure I understand what you want me to do :) Can you explain ?